examples/lsm-nice/lsm-nice-ebpf/src/main.rs

```diff
@@ -30,19 +31,20 @@ unsafe fn try_task_setnice(ctx: LsmContext) -> Result<i32, i32> {
     let p: *const task_struct = ctx.arg(0);
     let nice: c_int = ctx.arg(1);
     let ret: c_int = ctx.arg(2);
+    let global_pid: c_int = core::ptr::read_volatile(&PID);
 
-    // If previous eBPF LSM program didn't allow the action, return the
-    // previous error code.
+    info!(&ctx,
+          "The PID supplied to this program is: {}, with nice value {} and return value {}. Monitoring for changes in PID: {}",
+          (*p).pid, nice, ret, global_pid);
     if ret != 0 {
         return Err(ret);
     }
 
-    // Deny setting the nice value lower than 0 for the defined PID.
-    if (*p).pid == core::ptr::read_volatile(&PID) && nice < 0 {
+    if (*p).pid == global_pid && nice < 0 {
+        info!(&ctx, "Nice value is less than zero, returning error");
```

Comment:
can we remove this one? all the information is available (and present) in the previous log line.

In other words, let's log just once and have it contain everything needed.

Reply:
Sure